File: Java/2.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter first value: ");
        int first = in.nextInt();
        System.out.println("Enter second value: ");
        int second = in.nextInt();
        int sum = (first + second);
        int multip = (first * second);
        System.out.println("Sum: " + sum);
        System.out.println("Multiplication: " + multip);

    }
}

File: Python/2.py
def summa (first, second):
    return first + second

def multip (first, second):
    return first * second

def main():
    first = int(input("Enter first value: "))
    second = int(input("Enter second value: "))
    print(summa(first,second))
    print(multip(first,second))

if __name__ == "__main__":
    main()